src/clojure_lsp/api.clj

```diff
@@ -0,0 +1,15 @@
+(ns clojure-lsp.api
+  (:require
+    [clojure-lsp.internal-api :as internal-api]))
+
+(defn clean-ns!
+  "Organize ns form, removing unused requires/refers/imports and sorting
+  alphabetically.
+  Options should be passed as keyword arguments and it accepts:
+
+  `:project-root` a string representing the path to the project root.
```

Comment:
> `:project-root` a string representing the path to the project root.

Maybe accepting it as a File instead would invite people to do the right thing (namely, not hardcoding `/`s).

Might also be handy to `{:pre [(.exists the-file)]}` - APIs that fail-fast at the right level of abstraction tend to be friendlier

Reply:
Thank you! I was wondering if this would be a file or a string path indeed, clojure-lsp would accept both with or without the `/`, but it seems to makes sense come as a file indeed :)

Reply:
Done!